Sources/NIOSSL/TLSConfiguration.swift

```diff
@@ -439,7 +446,8 @@ extension TLSConfiguration {
                                 shutdownTimeout: .seconds(5),
                                 keyLogCallback: nil,
                                 renegotiationSupport: .none,
-                                additionalTrustRoots: [])
+                                additionalTrustRoots: [],
+                                sendCANameList: true)
```

Comment:
I'm inclined to say this shouldn't default to `true`: it's an unexpected change in behaviour. The correct default is `false`.